lms/djangoapps/grades/tasks.py

```diff
@@ -61,13 +62,41 @@ class _BaseTask(PersistOnFailureTask, LoggedTask):  # pylint: disable=abstract-m
 
 
 @task(base=_BaseTask)
-def compute_grades_for_course(course_key, offset, batch_size):
+def compute_grades_for_course_with_kwargs(course_key, offset, batch_size, **kwargs):
```

Comment:
Can you change the method name to `compute_grades_for_course_v2`?  That is the naming convention we have chosen for versioning our tasks.